components/templates/Books.tsx

```diff
@@ -24,6 +26,18 @@ const useStyles = makeStyles((theme) => ({
   title: {
     marginBottom: theme.spacing(2),
   },
+  linkedBook: {
+    // NOTE: 決め打ち (動画プレイヤーの高さに依存)
+    minHeight: 270,
```

Comment:
決め打ちではあるのですが値を盛りすぎました

```suggestion
    minHeight: 255,
```